src/App.ts

```diff
@@ -463,16 +481,20 @@ export default class App {
     }
 
     // Set respond() utility
-    if (respond !== undefined) {
-      listenerArgs.respond = respond;
+    if (body.response_url) {
+      listenerArgs.respond = async (response): Promise<any> => {
+        const postResponse = await this.axios.post(body.response_url, response);
```

Comment:
Related to above, `respond` does `await` here. This means `respond` may return a failure while the current `say` doesn't. Also, should we be consistent about when to apply App's  global error handler?

Reply:
Awesome catch, yes, I meant to remove the global handler catch from `say` as well. I'll update that.